projects/openmote-b/01bsp_openradio_rx/opentestbed_pdr.py

```diff
@@ -0,0 +1,252 @@
+import paho.mqtt.client as mqtt
```

Comment:
I guess this file is copied from another port.  If you didn't use it,  don't commit it.